### ISSUE TITLE
Introduce prettier and eslint to Origami.

### ISSUE THREAD

**Issue by fristonio** [Cloud-CV/Origami#144]
Currently, we have prettier and eslint as a dependency in `package.json` but they are not configured.

This issue is a child for the issue #132 We should fix the `package.json` first and then add the configuration for eslint and then run a `npm run lint:fix`(maybe after refactoring package.json) to fix lint issues in Origami.

This issue depends on https://github.com/Cloud-CV/Origami/issues/132

**Comment by uttu357** [comment]
Mention this issue in #145

**Comment by fristonio** [comment]
I think there should be a separate Pull Request for this. The main concern of this issue is fixing lint error in Origami using `npm run lint:fix` There are a lot of linting issues in Origami as of now and since it does not add any new feature or fix a bug it should be in a separate PR. You thoughts on this @uttu357

**Comment by uttu357** [comment]
Sure once #145 is merged you can make a separate PR fixing this.
